grid-demo/src/app/angular/lazy-loading-rows/lazy-loading-rows.component.html

```diff
@@ -1,3 +1,33 @@
+<clr-accordion>
+  <clr-accordion-panel [clrAccordionPanelOpen]="true">
+    <clr-accordion-title>Demo Explanation</clr-accordion-title>
+    <clr-accordion-content>
+      <p>
+        This demo allows for the datagrid to load rows lazily. At first all rows will show a loading indication, but
+        after scrolling to a row that is loading, the data will load in. The data will not attempt to load while the
+        datagrid is scrolling, but instead waits for a pause in the scrolling to load the data. Overtime, more and more
+        rows will be loaded until all rows all loaded. You can see how many rows are loaded in the bottom right corner
+        of the footer. This demo also uses virtual scrolling to ensure good performance.
+      </p>
+
+      <p>
```

Comment:
This one definitely needs some sort of A11Y callout or fallout.

Maybe something like "Accessibility: Pending A11Y review"?